### ISSUE TITLE
A11y/i18n: Aria label translations missing

### ISSUE THREAD

**Issue by samuelwei** [THM-Health/PILOS#2856]
**Describe the bug**
Most components have internationalised aria-labels. 

However, some components directly sourced from the PrimeVue UI-Library have their own aria-labels that aren’t explicitly set and therefore lack internationalisation.

**To Reproduce**

1. Change locale to German
2. Enable browser inspector
3. View pagination buttons on the room dashboard page
4. See aria-labels in english only

**Expected behavior**
All aria-labels should be properly internationalised

**Screenshots**
<img width="1268" height="544" alt="Image" src="https://github.com/user-attachments/assets/0c1f4c1d-070f-4faf-a538-9e627337a3ad" />
